python/kfserving/kfserving/kfmodels/sklearn.py

```diff
@@ -16,27 +16,40 @@
 import joblib
 import numpy as np
 import os
-from typing import List, Dict
+from typing import Dict
+from kfserving.kfmodels.kfmodel_types import KFModelTypes, MODEL_EXTENSIONS
 
 MODEL_BASENAME = "model"
-MODEL_EXTENSIONS = [".joblib", ".pkl", ".pickle"]
+MODEL_EXTENSIONS = MODEL_EXTENSIONS[KFModelTypes.Sklearn]
 
 
-class SKLearnModel(kfserving.KFModel): #pylint:disable=c-extension-no-member
+class SKLearnModel(kfserving.KFModel):  # pylint:disable=c-extension-no-member
     def __init__(self, name: str, model_dir: str):
         super().__init__(name)
         self.name = name
+        self.full_model_path = ""
         self.model_dir = model_dir
         self.ready = False
+        self._model = None
 
-    def load(self):
+    def load_from_model_dir(self):
         model_path = kfserving.Storage.download(self.model_dir)
         paths = [os.path.join(model_path, MODEL_BASENAME + model_extension)
                  for model_extension in MODEL_EXTENSIONS]
         model_file = next(path for path in paths if os.path.exists(path))
-        self._model = joblib.load(model_file) #pylint:disable=attribute-defined-outside-init
+        self._model = joblib.load(model_file)  # pylint:disable=attribute-defined-outside-init
         self.ready = True
 
+    def load_from_full_model_path(self):
```

Comment:
Do we really need to load a model from absolute path? Let's remove it, if we never will reach this code.
Also, consider change the interface to load_from_absolute_path(path)